templates/rails/files/.platform.app.yaml

```diff
@@ -42,8 +42,19 @@ mounts:
 hooks:
     # The build hook turns what is in source control into the deployable application.
     build: |
-        gem install bundler:2.0.2
-        bundle install --without development test
+        gem install --no-document bundler:2.1.4
+        bundle config set without 'development test'
+        bundle config deployment true
+        bundle config cache_all true
+        bundle config clean true
+        echo "Installing gems"
+        # when cache directory exists apply it to app so bundle use it
+        [ -d "$PLATFORM_CACHE_DIR/bundle-$PLATFORM_BRANCH" ] && \
```

Comment:
`$PLATFORM_BRANCH` is not available in the build hook.  It has to be a single pool for every branch.

Reply:
I see its only on deploy too bad thanks fixed